products/compute/terraform.yaml

```diff
@@ -623,6 +623,8 @@ overrides: !ruby/object:Provider::ResourceOverrides
           function: 'validateGCPName'
       secondaryIpRanges: !ruby/object:Provider::Terraform::PropertyOverride
         name: secondaryIpRange
+        unordered_list: true
+        default_from_api: true
```

Comment:
why this change?

Reply:
The problem comes from the fact that we're implicitly comparing against the returned value from the API - we wouldn't have any diff problems otherwise.